controllers/cluster_controller_test.go

```diff
@@ -3187,9 +3192,13 @@ var _ = Describe("cluster_controller", func() {
 				})
 
 				It("should include the cluster with the old default", func() {
-					deprecations, err := clusterReconciler.GetDeprecations(context.TODO())
-					Expect(err).NotTo(HaveOccurred())
-					Expect(len(deprecations)).To(Equal(1))
+					var deprecations []fdbtypes.FoundationDBCluster
+					Eventually(func() ([]fdbtypes.FoundationDBCluster, error) {
+						var err error
+						deprecations, err = clusterReconciler.GetDeprecations(context.TODO())
+						return deprecations, err
+					}, 5*time.Second).Should(HaveLen(1))
```

Comment:
The timeout `5*time.Second` can be removed here since it's the default.

```suggestion
					}).Should(HaveLen(1))
```